fragments/k8s/core/bundle.yaml

```diff
@@ -29,9 +28,8 @@ applications:
     to:
       - "1"
     options:
-      channel: 1.23/edge
+      channel: 1.24/edge
     expose: true
-    constraints: "cores=4 mem=4G root-disk=16G"
```

Comment:
removed the charm constraint since these are defined in the `machines` stanza below.